e2e/components/drafts/SelectPanel.test.ts

```diff
@@ -2,6 +2,12 @@ import {test, expect} from '@playwright/test'
 import {visit} from '../../test-helpers/storybook'
 import {themes} from '../../test-helpers/themes'
 
+const stories = await Promise.all([
```

Comment:
Didn't understand what happens when we import the stories?

Reply:
Hah, sorry about this! Was trying something out and forget to remove it lol.